README.rst

```diff
@@ -5,9 +5,9 @@ PyVO
     :target: http://www.astropy.org
     :alt: Powered by Astropy Badge
 
-.. image:: https://travis-ci.org/astropy/pyvo.svg
-    :target: https://travis-ci.org/astropy/pyvo
-    :alt: Travis Status
+.. image:: https://github.com/astropy/pyvo/workflows/CI/badge.svg?branch=master&event=schedule
```

Comment:
Why `event=schedule`?

Reply:
Because it reflects the (more or less current) status of the `master` branch. Where should we point it to?

Reply:
Maybe https://github.com/astropy/pyvo/workflows/CI/badge.svg?branch=master is enough so it picks up merge commit runs too?